adm_program/installation/index.php

```diff
@@ -9,8 +9,24 @@
  ***********************************************************************************************
  */
 
+
+$rootPath = dirname(__DIR__, 2);
+
 // check if installation is necessary
-if (is_file('../../adm_my_files/config.php')) {
+if (is_file($rootPath . '/adm_my_files/config.php')) {
```

Comment:
the second check here is required to make a difference between normal installation and docker installation.
With a Docker installation, it is possible that a config.php exists, but the database has not yet been initialized.